### PR TITLE
Scan: Revamp delegate modal

### PR DESCRIPTION
**Pull request by Pitchanai**
Fixed: #2732

## Implementation details
- Revamp delegate modal

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [x] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: scan/src/components/EnhanceTxInput.re [hunk 1]
```diff
@@ -99,8 +75,9 @@ let make =
   };
 
   <div className=Styles.container>
-    <Text value=msg size=Text.Lg spacing={Text.Em(0.03)} nowrap=true block=true />
-    <div className={Styles.rightContainer(width)}>
+    <Text value=msg size=Text.Md weight=Text.Medium nowrap=true block=true />
+    <VSpacing size=Spacing.sm />
+    <div className=Styles.rightContainer>
```
Comment (pzshine) [inline scan/src/components/EnhanceTxInput.re]: Could you change it `CssHelper.flexbox()`?

File: scan/src/components/EnhanceTxInput.re [hunk 2]
```diff
@@ -138,8 +115,29 @@ module Loading = {
...
+  };
+};
+
+module Loading2 = {
+  [@react.component]
+  let make = (~msg, ~useMax=false, ~code=false, ~placeholder) => {
+    <div className=Styles.container>
+      <Text value=msg size=Text.Md weight=Text.Medium nowrap=true block=true />
+      <VSpacing size=Spacing.sm />
+      <div className=Styles.rightContainer>
```
Comment (pzshine) [inline scan/src/components/EnhanceTxInput.re]: Change it to `CssHelper.flexbox()`

File: scan/src/components/modal/submitTx/SubmitTxModal.re [hunk 3]
```diff
@@ -4,21 +4,25 @@ module Styles = {
...
+  let modalTitle = style([paddingBottom(`px(40))]);
 
   let rowContainer =
-    style([display(`flex), alignItems(`center), justifyContent(`spaceBetween)]);
+    style([
+      display(`flex),
+      alignItems(`center),
+      justifyContent(`spaceBetween),
+      paddingBottom(`px(24)),
+    ]);
```
Comment (evilpeach) [inline scan/src/components/modal/submitTx/SubmitTxModal.re]: remove?

**Review by pzshine — CHANGES_REQUESTED** [review]

File: scan/src/components/modal/submitTx/SubmitTxModal.re [hunk 4]
```diff
@@ -178,7 +157,7 @@ module SubmitTxStep = {
             };
           ();
         }}>
-        <Text value="Next" weight=Text.Bold size=Text.Md />
+        <Text value="Next" weight=Text.Medium size=Text.Lg />
       </button>
```
Comment (pzshine) [inline scan/src/components/modal/submitTx/SubmitTxModal.re]: Could we change to `<Button/>` component?

**Review by pzshine — CHANGES_REQUESTED** [review]

File: scan/src/components/modal/submitTx/SubmitTxModal.re (hunk 4 above)
Comment (Pitchanai) [reply scan/src/components/modal/submitTx/SubmitTxModal.re]: @pzshine I've changed to `<Button/>` but have to fix cypress testing later.

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Pull request merged**